src/bin/pg_autoctl/monitor_pg_init.c

```diff
@@ -223,9 +226,11 @@ monitor_install(PostgresSetup pgSetupOption, bool checkSettings)
 	 * Now allow nodes on the same network to connect to pg_auto_failover
 	 * database.
 	 */
-	if (!pghba_enable_lan_cidr(&postgres.sqlClient, HBA_DATABASE_DBNAME,
-							  PG_AUTOCTL_MONITOR_DBNAME,
-							  PG_AUTOCTL_MONITOR_USERNAME, "trust", NULL))
+	if (!pghba_enable_lan_cidr(&postgres.sqlClient,
+							   HBA_DATABASE_DBNAME,
+							   PG_AUTOCTL_MONITOR_DBNAME,
+							   (char *)nodename,
```

Comment:
instead of casting here, can we make `pghba_enable_lan_cidr` a `const`? I don't see any reason to change the nodename by that function.
seeing the definition below, I think all `char *` should be `const`s

Reply:
Good point, done that way now.